package.json

```diff
@@ -30,6 +30,7 @@
     "crypto-js": "3.1.6",
     "lodash": "3.10.1",
     "node-uuid": "1.4.7",
+    "pako": "1.0.3",
```

Comment:
If we want things to work in the browser too, we can add a browserify/webpack rule